xcode/Subconscious/Shared/Models/EntryStub.swift

```diff
@@ -16,6 +16,7 @@ struct EntryStub:
     CustomDebugStringConvertible,
     Codable
 {
+    let did: Did
```

Comment:
All `EntryStub`s now come with a `did`. This is always available when reading from the DB and means we can trivially work out which user profile should be associated with a given entry.